aws/network-primitives/scripts/create_bootstrap_node_compose_file.sh

```diff
@@ -21,58 +27,73 @@ services:
       - /etc/os-release:/host/etc/os-release:ro
 
   dsn-bootstrap-node:
-    image: ghcr.io/\$NODE_ORG/bootstrap-node:\$NODE_TAG
+    image: ghcr.io/\${NODE_ORG}/bootstrap-node:\${NODE_TAG}
     restart: unless-stopped
     environment:
       - RUST_LOG=info
     ports:
-      - "\$DSN_LISTEN_PORT:50000"
+      - "30533:30533"
     command:
       - start
-      - \$DSN_NODE_KEY
-      - /ip4/0.0.0.0/tcp/50000
+      - \${DSN_NODE_KEY}
+      - /ip4/0.0.0.0/tcp/30533
       - --protocol-version
       - \${GENESIS_HASH}
```

Comment:
Shouldn't the genesis hash be here  since i remember we are passing an empty value above ?

Reply:
We pass an empty value first, and then attain it when launching the node. Once launched it's populated in `main.tf`